translations/de.json

```diff
@@ -3,6 +3,7 @@
     "de": "Deutsch"
   },
   "panel": {
+    "config": "Konfiguration",
```

Comment:
Maybe change translation to 'Einstellungen'

Reply:
both have the same meaning, it's `configuration` vs `settings`- I hope we get a platform for translations soon so we can discuss/vote such things :)